spark/unitycatalog/src/test/java/com/sparkuctest/UnityCatalogSupport.java

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.sparkuctest;
+
+import io.unitycatalog.client.ApiClient;
+import io.unitycatalog.client.api.CatalogsApi;
+import io.unitycatalog.client.api.SchemasApi;
+import io.unitycatalog.client.model.CreateCatalog;
+import io.unitycatalog.client.model.CreateSchema;
+import io.unitycatalog.server.UnityCatalogServer;
+import io.unitycatalog.server.utils.ServerProperties;
+import org.apache.log4j.Logger;
+import org.apache.spark.SparkConf;
+import org.junit.After;
+import org.junit.Before;
```

Comment:
`org.junit.After` or `org.junit.jupiter.api` ?  Which is the preference ?  In unitycatalog, we use the later one.

Reply:
The first one is junit4 style. The latter is Junit5 style. We should use the latter.

Reply:
updating this module to use junit5